exporter/collector/logs.go

```diff
@@ -215,9 +223,12 @@ func (l *LogsExporter) PushLogs(ctx context.Context, ld plog.Logs) error {
 	return nil
 }
 
-func (l logMapper) createEntries(ld plog.Logs) ([]*logpb.LogEntry, error) {
+func (l logMapper) createEntries(ld plog.Logs) (map[string][]*logpb.LogEntry, error) {
+	// if destination_project_quota is enabled, projectMapKey will be the name of the project for each batch of entries
+	// otherwise, we can mix project entries for more efficient batching and store all entries in a single list
+	projectMapKey := "google-destination-quota-disabled"
```

Comment:
could this just always be the destination project id instead of this string?  Then other functions could assume the key of `map[string][]*logpb.LogEntry` is the destination project?

Reply:
if destination quota is disabled, entries can be unsorted (wrt project), so using a default key packs them into a single list that is used by the other functions, and the other functions ignore the key (no need to set any project id in the request)

with destination quota enabled, this key does become the destination project and each entry gets sorted into its project's list. then the other function uses that key to set the x-goog header

Reply:
Ah, got it.  I'd prefer using the empty string, as someone might think the constant is important...  But not a big deal.

Reply:
Oh empty string is much better